atom/browser/node_debugger.cc

```diff
@@ -30,10 +30,16 @@ void NodeDebugger::Start() {
   std::vector<std::string> args;
   for (auto& arg : base::CommandLine::ForCurrentProcess()->argv()) {
 #if defined(OS_WIN)
-    args.push_back(base::UTF16ToUTF8(arg));
+    const std::string nice_arg = base::UTF16ToUTF8(arg);
 #else
-    args.push_back(arg);
+    const std::string& nice_arg = arg;
 #endif
+    // Stop handling arguments after a "--" to be consistent with Chromium
+    // This is here for security reasons, do not remove
```

Comment:
Please be specific about "security reasons". Be clear in the comment under what circumstances untrusted arguments can arrive in this code.